src/admin/content/content.service.ts

```diff
@@ -437,4 +464,121 @@ export class ContentService {
 	): ContentBlockConfig[] {
 		return omitByDeep(blockConfigs, key => String(key).endsWith(RichEditorStateKey));
 	}
+
+	/**
+	 * Find name that isn't a duplicate of an existing name of a content page of this user
+	 * eg if these content pages exist:
+	 * copy 1: test
+	 * copy 2: test
+	 * copy 4: test
+	 *
+	 * Then the algorithm will propose: copy 3: test
+	 * @param copyPrefix
+	 * @param copyRegex
+	 * @param existingTitle
+	 *
+	 * @returns Potential title for duplicate content page.
+	 */
+	public static getCopyTitleForContentPage = async (
```

Comment:
mss nog een todo toevoegen om deze code generieker te maken zodat ze in de collectionService en in de contentService kan gebruikt worden.